### PR TITLE
Added test for preserving child node order on copy

### PR DESCRIPTION
**Pull request by dantleech**
Test for preserving child order on copy, see https://github.com/jackalope/jackalope-doctrine-dbal/pull/333

### CODE REVIEW

File: tests/Writing/CopyMethodsTest.php [hunk 1]
```diff
@@ -79,6 +79,24 @@ public function testWorkspaceCopy()
...
+
+        $node = $this->session->getNode($src);
+        $node->orderBefore('three', 'one');
+        $this->session->save();
+        $this->assertEquals($expected, iterator_to_array($node->getNodeNames()));
+
+        $this->ws->copy($src, $dst);
+
+        $node = $this->session->getNode($dst);
+        $this->assertEquals($expected, iterator_to_array($node->getNodeNames()));
```
Comment (dantleech) [inline tests/Writing/CopyMethodsTest.php]: Currently fails here on at least Jackalope DBAL as expected.

**Comment by dbu** [comment]
@dantleech would you mind testing this with the current dbal and jackrabbit clients to make sure the test is correct the issue fixed in dbal? then we can merge this and update the tests

**Comment by dantleech** [comment]
yep, works on Jackrabbit, failed on DBAL before the merge and passes after.

**Comment by dbu** [comment]
excellent, thanks for the help!

**Pull request merged**